doc/glossary.rst

```diff
@@ -149,6 +149,12 @@ General Concepts
         introduces changes that are not backwards compatible, these are known
         as software regressions.
 
+    callable
+        A function, class or an object which implements the ``__call__``
+        method; anything that returns True when the argument of
+        `https://docs.python.org/3/library/functions.html#callable
```

Comment:
I think that you wanted to do the opposit: callable() <https://docs.python.org/3/library/functions.html#callable>_

Reply:
Of course!